rocky/onboarding/forms.py

```diff
@@ -36,3 +36,17 @@ class OnboardingSetClearanceLevelForm(forms.Form):
             },
         ),
     )
+
```

Comment:
I'd rather see a solution where all the OOI-add forms do not display their optional selection fields when there are no options for such a field. There is a separate issue for that available here:
https://github.com/minvws/nl-kat-coordination/issues/2139 

following that route would also negate adding more code for this specific onboarding form.

Reply:
Yes, I will resolve this in another PR, it will be then applicable in the Create object form. I think for the onboarding a simple URL field is enough. As you can see that I need to reformat the form too much using init, while I can simple create a new object.

Reply:
I agree, and that should be fixed in the report flow first.

Adding an OOI itself should also be made simpler by not showing optional fields that have no options. that's where the other issue (#2139) comes in.